fix: clarification about metadata in subordiante statements (#26)
peppelinux/federation-wallet: openid-federation-wallet-1_0.md

> The key difference between `metadata` and `metadata_policy` is that metadata directly affects only the Immediate Subordinate Entity, while `metadata_policy` impacts the configuration of all Subordinate Entities along a Trust Chain, as defined in Sections 5 and 6.1 of [@!OpenID.Federation].

Author: ```suggestion
The key difference between `metadata` and `metadata_policy` is that metadata directly affects only the Immediate Subordinate Entity, while `metadata_policy` impacts the configuration of all Subordinate Entities along a Trust Chain, as defined in Sections 3, 5 and 6.1 of [@!OpenID.Federation].
```

The key difference between `metadata` and `metadata_policy` is that metadata directly affects only the Immediate Subordinate Entity, while `metadata_policy` impacts the configuration of all Subordinate Entities along a Trust Chain, as defined in Sections 5 and 6.1 of [@!OpenID.Federation].

> This distinction positions the `metadata` parameter as an ideal tool for federation authorities managing Immediate Subordinates registrations and needing to sanitize Leaves configurations in an arbitrary way. The Trust Anchor (TA) and Intermediate (INT) sanitize an Entity Configuration during technical tests and finalize it by setting specific metadata parameters.

Author: ```suggestion
This distinction positions the `metadata` parameter as an ideal tool for federation authorities managing Immediate Subordinates registrations, as Leaves, and needing to sanitize their configurations in an arbitrary way. The Trust Anchor (TA) and Intermediate (INT) sanitize an Entity Configuration during technical tests and finalize it by setting specific metadata parameters.
```

> This distinction positions the `metadata` parameter as an ideal tool for federation authorities managing Immediate Subordinates registrations, as Leaves, and needing to sanitize their configurations in an arbitrary way. The Trust Anchor (TA) and Intermediate (INT) sanitize an Entity Configuration during technical tests and finalize it by setting specific metadata parameters.

Author: ```suggestion
The `metadata` parameter allows federation authorities, managing Immediate Subordinates registrations, to sanitize their configurations in an arbitrary way, without necessarly use the policy language used with the `metadata_policy` parameter . The Trust Anchor (TA) and Intermediate (INT) sanitize an Entity Configuration during technical tests and finalize it by setting specific metadata parameters. The `metadata` parameter value updates all the matching part of the `metadata` json object contained within the Subordinate Entity Configuration.
```

> The `metadata` parameter allows federation authorities managing Immediate Subordinates' registrations to sanitize their configurations in an arbitrary way, without necessarily using the policy language in the `metadata_policy` parameter. The Trust Anchor (TA) and Intermediate (INT) sanitize an Entity Configuration and finalize it by setting specific metadata parameters. The `metadata` parameter value updates all the matching parts of the `metadata` JSON object contained within the Subordinate Entity Configuration.

Author: ```The `metadata` parameter value updates all the matching parts of the `metadata` JSON object```

@selfissued we should say here that it updated if any matching, otherwise it add the unmatching parameters.

> The `metadata` parameter allows federation authorities managing Immediate Subordinates' registrations to sanitize their configurations in an arbitrary way, without necessarily using the policy language in the `metadata_policy` parameter. The Trust Anchor (TA) and Intermediate (INT) sanitize an Entity Configuration and finalize it by setting specific metadata parameters. The `metadata` parameter value updates all the matching parts of the `metadata` JSON object contained within the Subordinate Entity Configuration.

Reviewer: ```suggestion
```

Unnecessary

Reviewer: Section 3 of the Federation spec says this in the "metadata" definition:

> An Immediate Superior MAY provide selected or all metadata parameters for an Immediate Subordinate, by using the metadata claim in a Subordinate Statement. When metadata is used in a Subordinate Statement, it applies only to those Entity Types that are present in the subject's Entity Configuration. Furthermore, the metadata applies only to the subject of the Subordinate Statement and has no effect on the subject's Subordinates. Metadata parameters in a Subordinate Statement have precedence and override identically named parameters under the same Entity Type in the subject's Entity Configuration. If both metadata and metadata_policy appear in a Subordinate Statement, then the stated metadata MUST be applied before the metadata_policy, as described in [Section 6.1.4.2](https://openid.github.io/federation/main.html#metadata_policy_application).

We may want to cite that, rather than repeat the equivalent here.

> The `metadata` parameter allows federation authorities, managing Immediate Subordinates registrations, to sanitize their configurations in an arbitrary way, without necessarly use the policy language used with the `metadata_policy` parameter . The Trust Anchor (TA) and Intermediate (INT) sanitize an Entity Configuration during technical tests and finalize it by setting specific metadata parameters. The `metadata` parameter value updates all the matching part of the `metadata` json object contained within the Subordinate Entity Configuration.

Reviewer: ```suggestion
The `metadata` parameter allows federation authorities managing Immediate Subordinates' registrations to sanitize their configurations in an arbitrary way, without necessarily using the policy language in the `metadata_policy` parameter. The Trust Anchor (TA) and Intermediate (INT) sanitize an Entity Configuration and finalize it by setting specific metadata parameters. The `metadata` parameter value updates all the matching parts of the `metadata` JSON object contained within the Subordinate Entity Configuration.
```
